L1TriggerConfig/L1TConfigProducers/src/L1TMuonOverlapFwVersionOnlineProd.cc

```diff
@@ -0,0 +1,147 @@
+#include <iostream>
+#include <fstream>
+
+#include "CondTools/L1TriggerExt/interface/L1ConfigOnlineProdBaseExt.h"
+#include "CondFormats/L1TObjects/interface/L1TMuonOverlapFwVersion.h"
+#include "CondFormats/DataRecord/interface/L1TMuonOverlapFwVersionRcd.h"
+#include "CondFormats/DataRecord/interface/L1TMuonOverlapFwVersionO2ORcd.h"
+#include "L1Trigger/L1TCommon/interface/TriggerSystem.h"
+#include "L1Trigger/L1TCommon/interface/XmlConfigParser.h"
+#include "OnlineDBqueryHelper.h"
+
+#include "xercesc/util/PlatformUtils.hpp"
+using namespace XERCES_CPP_NAMESPACE;
+
+class L1TMuonOverlapFwVersionOnlineProd : public L1ConfigOnlineProdBaseExt<L1TMuonOverlapFwVersionO2ORcd, L1TMuonOverlapFwVersion> {
+private:
+  const bool transactionSafe;
+  const edm::ESGetToken<L1TMuonOverlapFwVersion, L1TMuonOverlapFwVersionRcd> baseSettings_token;
+
+public:
+  std::unique_ptr<const L1TMuonOverlapFwVersion> newObject(const std::string& objectKey, const L1TMuonOverlapFwVersionO2ORcd& record) override;
+
+  L1TMuonOverlapFwVersionOnlineProd(const edm::ParameterSet&);
+  ~L1TMuonOverlapFwVersionOnlineProd(void) override {}
```

Comment:
```suggestion
  ~L1TMuonOverlapFwVersionOnlineProd(void) override = default;
```
This is the new suggestion from the framework team. Please apply it elsewhere too when this comes up

Reply:
@michalszleper you havent changed this yet

Reply:
I changed to override = default, but only in the files which are part of this PR.